.github/workflows/misc.yml

```
@@ -32,3 +32,22 @@ jobs:
           path: out/doc
       - name: Test
         run: NODE=$(command -v node) make test-doc-ci TEST_CI_ARGS="-p actions"
+
+  build-lto:
+    if: github.event.pull_request.draft == false
```

```suggestion
    if: github.event.pull_request.draft == false and startsWith(github.ref, 'refs/tags/v')
```

@targos Maybe like this ?

---

Another possibility is to create a new workflow file for this. I'd also be fine with a schedule instead of filtering by release tag.

---

@targos PTAL.